src/js/services/storageService.js

```diff
@@ -78,6 +78,14 @@ angular.module('copayApp.services')
     //
     // UPGRADING STORAGE
     //
+    // TODO: prior to upgrading storage a backup of all storage should be captured and made available
```

Comment:
Could you explain this more? We cannot merge this to master, if this is a requirement.